install/installer/docs/overview.md

```diff
@@ -0,0 +1,589 @@
+<p align="center">
+  <a href="https://www.gitpod.io">
+    <img src="https://raw.githubusercontent.com/gitpod-io/gitpod/master/components/dashboard/src/icons/gitpod.svg" height="60">
+    <h3 align="center">Gitpod</h3>
+  </a>
+  <p align="center">Always ready-to-code.</p>
+</p>
+
+# Installer
+
+The best way to get started with Gitpod is by using our recommended & default installation method [described in our documentation](https://www.gitpod.io/docs/self-hosted/latest/getting-started). In fact, our default installation method actually wraps this installer into a UI that helps you manage, update and configure Gitpod in a streamlined way. This document describes how to use the installer directly.
+
+> The installer is an internal tool and as such not expected to be used by those external to Gitpod.
+
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-908a85?logo=gitpod)](https://gitpod.io/from-referrer/)
```

Comment:
nit: But I wonder if we need this badge anymore in this docs file? It makes sense in the README to open a gitpod instance with that code but not here? 🤔